azure-servicebus/src/main/java/com/microsoft/azure/servicebus/MessageReceiver.java

```diff
@@ -679,9 +678,19 @@ public Instant renewMessageLock(IMessage message) throws InterruptedException, S
         return Utils.completeFuture(this.renewMessageLockAsync(message));
     }
 
+    @Override
+    public Instant renewMessageLock(UUID lockToken)  throws InterruptedException, ServiceBusException {
+        return Utils.completeFuture(this.renewMessageLockAsync(lockToken));
+    }
+
     //	@Override
     public Collection<Instant> renewMessageLockBatch(Collection<? extends IMessage> messages) throws InterruptedException, ServiceBusException {
```

Comment:
>renewMessageLockBatch [](start = 31, length = 21)

Create an async method of this. Write the sync method as Util.completeFuture() version of the async method.